common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt

```diff
@@ -160,6 +121,18 @@ data class BrokerData(val packageName : String,
             AuthenticationConstants.Broker.LTW_APP_SHA512_DEBUG_SIGNATURE
         )
 
+        @JvmStatic
+        val accountManagerBrokers: Set<String> =
+            Collections.unmodifiableSet(object : HashSet<String>() {
+                init {
+                    add(AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_PACKAGE_NAME)
+                    add(AuthenticationConstants.Broker.COMPANY_PORTAL_APP_PACKAGE_NAME)
+                    add(AuthenticationConstants.Broker.BROKER_HOST_APP_PACKAGE_NAME)
+                    add(AuthenticationConstants.Broker.MOCK_AUTH_APP_PACKAGE_NAME)
+                    add(AuthenticationConstants.Broker.MOCK_CP_PACKAGE_NAME)
```

Comment:
Should these be added behind the check sShouldTrustDebugBrokers

Reply:
It already is (but as a set of BrokerData, not just names like this one object).